release/packbeam.in

```diff
@@ -19,4 +19,10 @@
 # SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
 #
 
-exec PACKBEAM_INSTALL_LOCATION/bin/packbeam "$@"
+root_dir="$(cd $(dirname $0)/.. && pwd)"
+pkg_root="${root_dir}/atomvm_packbeam"
+
+PATH="${pkg_root}/erts-{{ release_erts_version }}/bin:${PATH}"
```

Comment:
What ends up in the `${pkg_root}/errs-{{ release_erts_version }}/bin` directory? And why does it need to be exported?